CHANGELOG.next.asciidoc

```diff
@@ -601,6 +601,19 @@ same journal. {pull}18467[18467]
 - Add awsfargate module task_stats metricset to monitor AWS ECS Fargate. {pull}22034[22034]
 - Add connection and route metricsets for nats metricbeat module to collect metrics per connection/route. {pull}22445[22445]
 - Add unit file states to system/service {pull}22557[22557]
+- Add io.ops in fields exported by system.diskio. {pull}22066[22066]
```

Comment:
Please fix the changelog.

Reply:
Good catch! Thank you for noticing! Fixed!